.circleci/config.yml

```diff
@@ -55,26 +55,24 @@ jobs:
             . gwenv/bin/activate
             pip install -U pip setuptools wheel
             # Install pre-geowombat libraries
-            # This shouldn't be necessary with pyproject.toml
-            pip install Cython numpy
+            # This shouldn't be necessary with pyproject.toml but we need numpy for GDAL
+            pip install -U numpy
             pip install GDAL==$GDAL_VERSION
             # Install GeoWombat
-            pip install .
-            # We should be able to install these as extras (i.e., .[ml])
-            pip install sklearn-xarray rtree
+            pip install .[perf,ml]
```

Comment:
We need to remember to update the readme about these changes

Reply:
Good call